server.js

```diff
@@ -20,9 +30,42 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Welcome to Happy Thoughts API by Jihyun')
+})
+
+app.get('/GET', async (req, res) => {
```

Comment:
You misunderstood the requirements( `get /`). this line should be: `app.get('/', async (req, res) => {`